src/main/java/Mediell/Task.java

```diff
@@ -0,0 +1,61 @@
+package Mediell;
+
+import java.util.Objects;
+
+/** Represents a Task. */
+public class Task {
+    private String taskName;
+    private boolean completed;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        completed = false;
```

Comment:
nit: Would `this.completed` be better as it is standardised with `this.taskName`?